thirdweb-dev/js
# [react] Add switchNetworkBtnTitle prop in ConnectWallet and Web3Button

## Problem solved

Adding a prop named `switchNetworkBtnTitle` that can be used to set a custom `Switch Network` text for the `ConnectWallet` component whenever the condition show the `SwitchNetworkButton` button is met.

## Changes made

- [ x] Public API changes: list the public API changes made if any

## How to test

- Create a new Thirdweb app with React or a React framework.
- Add the `ThirdwebProvider` at the root level or to a component to test and set any activeChain.
- Import the `ConnectWallet` component in any component that's inside the ThirdWeb Provider.
- Set a value for `switchNetworkBtnTitle` prop in the `ConnectWallet` component.
- Set `switchToActiveChain` prop to true for the `ConnectWallet` component.
- Run the local dev server and once the app is running, connect your wallet using the `ConnectButton` and switch to a different network from the `activeChain` set for the `ThirdwebProvider`.
- Verify that the `ConnectWallet` changed the `SwitchNetworkButton` and has the value of `switchNetworkBtnTitle` prop as the text.

- [ ] Automated tests: link to unit test file
- [ x] Manual tests: step by step instructions on how to test
